c-api/examples/wasm/index.html

```diff
@@ -0,0 +1,60 @@
+<h2>ttf-parser in WebAssembly</h2>
+<p><small>(supports font files drag and drop)</small></p>
+<p><span id="fileName">TTC.ttc</span>:</p>
+<p><code>ttfp_fonts_in_collection():</code> <code id="fontsInCollection"></code></p>
```

Comment:
wished `ttfp_fonts_in_collection` could return 1 for non ttc fonts and 0 for non fonts, that sounds more natural to me.

Reply:
I will think about it. The rust api is more clear since we have `Option`.